applications/GeoMechanicsApplication/tests/cpp_tests/test_T_micro_climate_flux_condition.cpp

```diff
@@ -435,12 +417,10 @@ KRATOS_TEST_CASE_IN_SUITE(CalculateLocalSystemForThermalMicroClimateCondition3D6
     p_condition->CalculateLocalSystem(lhs_matrix, rhs_vector, r_model_part.GetProcessInfo());
 
     auto expected_lhs_matrix = Matrix{6, 6, 0.0};
-    expected_lhs_matrix <<= 1.95146, -0.975729, -0.975729, 0.975729, -1.95146, 0.975729,
-            -0.975729, 1.95146, -0.975729, 0.975729, 0.975729, -1.95146,
-            -0.975729, -0.975729, 1.95146, -1.95146, 0.975729, 0.975729,
-            0.975729, 0.975729, -1.95146, 10.733, 7.80583, 7.80583,
-            -1.95146, 0.975729, 0.975729, 7.80583, 10.733, 7.80583,
-            0.975729, -1.95146, 0.975729, 7.80583, 7.80583, 10.733;
+    expected_lhs_matrix <<= 1.95146, -0.975729, -0.975729, 0.975729, -1.95146, 0.975729, -0.975729,
```

Comment:
This might be nice to revert and add the tags to ignore this for formatting

Reply:
Done.